jvb/src/main/kotlin/org/jitsi/videobridge/octo/config/OctoConfig.kt

```diff
@@ -62,15 +62,15 @@ class OctoConfig private constructor() {
     /**
      * This is deprecated and only used in octo v1.
      */
-    val bindAddress: String by config {
+    val bindAddress: String? by optionalconfig {
         "bind address from legacy config" { legacyBindAddress!! }
```

Comment:
Should this still have the `!!` here?

Reply:
Yes, the function requires a `String`. It should catch the NPE then rethrow a config exception, causing it skip to the next one. But I'll run some tests to verify.